### DISCUSSION TITLE
Render "minimum" and "exclusiveMinimum" restrictions

### DISCUSSION THREAD

**Discussion by Amorim33** [scalar/scalar#1956] [Ideas]
## Description

Scalar does not render labels for `minimum` and `exclusiveMinimum` properties. 

Scalar example:
![image](https://github.com/scalar/scalar/assets/42624869/8faa91ac-bbe7-4788-87ce-7cc6289a1156)

Redoc example:
![image](https://github.com/scalar/scalar/assets/42624869/3452a85f-3e06-496e-b406-8a1b36524dcb)

## OpenAPI Properties

```json
"balance": {
  "type": "integer",
  "minimum": 0,
  "exclusiveMinimum": true,
  "description": "Saldo devedor em centavos."
},
"remainingTerm": {
  "type": "integer",
  "minimum": 0,
  "description": "Prazo remanescente em meses."
},
```

## Reference
https://swagger.io/docs/specification/data-models/data-types/

> Not totally sure but probably there are work to do with "Minimum", "Maximum" and "Multiples", I am totally available to help you with these implementations 😊

**Comment by hanspagel** [comment]
Yes, we forgot about them! I’ve created an issue: https://github.com/scalar/scalar/issues/1986

If you’d like to pick this up: NICE. Let me know what you need to get started. If you don’t have the capacity, we’ll be able to tackle it, too. ✌️

**Reply by Amorim33** [reply]
Nice!!
I'm going to work on this tomorrow! 🙏
